Assets/MRTK/Providers/WindowsMixedReality/Shared/Editor/ConfigurationChecker/Microsoft.MixedReality.Toolkit.Providers.WindowsMixedReality.ConfigurationChecker.asmdef

```diff
@@ -1,16 +0,0 @@
-{
-    "name": "Microsoft.MixedReality.Toolkit.Providers.WindowsMixedReality.ConfigCheck",
-    "references": [
-        "Microsoft.MixedReality.Toolkit.Editor.Utilities"
```

Comment:
Not this assembly, but the others that have their file names change (couldn't figure out how to comment on a renamed but not changed file):

I believe Unity serializes these references by file name, not by actual assembly name. It'd be good to make sure none of the renamed Editor assemblies are referenced anywhere else.

Reply:
Actually I think I'm wrong here. I thought I had run into issues when file names changed, but I'm not seeing that at the moment.